src/main/java/org/folio/innreach/batch/contribution/IterationEventReaderFactory.java

```diff
@@ -69,7 +83,18 @@ private Deserializer<String> keyDeserializer() {
   }
 
   private static UUID getJobId(ConsumerRecord<String, InstanceIterationEvent> rec) {
+    // TODO Comment this back in after testing
     return UUID.fromString(new String(rec.headers().lastHeader(ITERATION_JOB_ID_HEADER).value()));
+//    return UUID.fromString("fa34246b-86a6-4743-8d20-f368ef3242d7");
+  }
+
+  public KafkaUtil createKafkaConsumer() {
```

Comment:
Let's rename this method to `createInitialContributionConsumerContainer` since it can be more than one and I'm recommending a rename of `KafkaUtil` to be more in line with our purposes.